From: =?utf-8?q?R=C3=A9mi_Palancher?= <remi@rezib.org>
Date: Tue, 11 Aug 2020 12:43:16 +0000
Subject: Increase sacctmgr buffer size

This patch significantly increases the size of the buffer for reading dumps
with sacctmgr load. Without this patch, sacctmgr is not able to handle lines
with more than 4096 which is too small in some cases.
---
 src/sacctmgr/file_functions.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/sacctmgr/file_functions.c b/src/sacctmgr/file_functions.c
index 78ff210..e51d5c0 100644
--- a/src/sacctmgr/file_functions.c
+++ b/src/sacctmgr/file_functions.c
@@ -42,7 +42,7 @@
 #include "src/common/strlcpy.h"
 #include "src/common/uid.h"
 
-#define BUFFER_SIZE 4096
+#define BUFFER_SIZE 524288
 
 typedef struct {
 	slurmdb_admin_level_t admin;
